src/base/utils/misc.cpp

```diff
@@ -620,8 +622,50 @@ QString Utils::Misc::zlibVersionString()
 }
 
 #ifdef Q_OS_WIN
+Path Utils::Misc::windowsSystemPath()
+{
+    static const Path path = []() -> Path
+    {
+        WCHAR systemPath[MAX_PATH] = {0};
+        GetSystemDirectoryW(systemPath, sizeof(systemPath) / sizeof(WCHAR));
+        return Path(QString::fromWCharArray(systemPath));
+    }();
+    return path;
+}
+#endif // Q_OS_WIN
+
+#if defined(Q_OS_MACOS) || defined(Q_OS_WIN)
 bool Utils::Misc::applyMarkOfTheWeb(const Path &file, const QString &url)
```

Comment:
Should we use some abstract name instead? Which will simply convey its meaning, and not the name of any particular implementation.

Reply:
> I thought of it but no good name so far.

OK, forget it.